src/App.tsx

```diff
@@ -1,56 +1,97 @@
-import React from 'react';
+/* eslint-disable jsx-a11y/control-has-associated-label */
+/* eslint-disable operator-linebreak */
+/* eslint-disable no-useless-concat */
```

Comment:
oh, that's not for good. Were there no any other ways to overcome this linting error?